dynamicweb/settings/base.py

```diff
@@ -462,17 +468,20 @@ def env(env_name):
 
 # EMAIL MESSAGES
 REGISTRATION_MESSAGE = {'subject': "Validation mail",
-                        'message': 'Thank You for registering for account on Digital Glarus.\nPlease verify Your account under following link http://{host}/en-us/digitalglarus/login/validate/{slug}',
+                        'message': 'Thank You for registering for account on Digital Glarus.\n'
+                                   'Please verify Your account under following link '
+                                   'http://{host}/en-us/digitalglarus/login/validate/{slug}',
                         }
 STRIPE_API_PRIVATE_KEY = env('STRIPE_API_PRIVATE_KEY')
 STRIPE_API_PUBLIC_KEY = env('STRIPE_API_PUBLIC_KEY')
 
 DEBUG = True
 
-if DEBUG:
-    from .local import *
-else:
-    from .prod import *
+# not used
+# if DEBUG:
+#     from .local import *
+# # else:
+# #     from .prod import *
 
```

Comment:
These values are in fact still used (though in a wrong way currently). I think we should still keep them.